Android/ExposureNotification/source/PlayServicesNearby/Additions/ExposureNotificationClient.cs

```diff
@@ -1,45 +1,42 @@
-﻿using System;
+﻿using Android.Runtime;
+using System;
 using System.Collections.Generic;
-using System.Linq;
 using System.Threading.Tasks;
-using Android.Content;
-using Android.Hardware;
-using Android.Runtime;
 using AndroidTask = Android.Gms.Tasks.Task;
 
 namespace Android.Gms.Nearby.ExposureNotification
 {
 	public partial interface IExposureNotificationClient
 	{
-		public Task Start(ExposureConfiguration config)
+		public Task StartAsync(ExposureConfiguration config)
```

Comment:
Added the Async suffix... Is this normal? Or should it go?

Reply:
I'm not fussed either way.